code/modules/vehicles/wheelchair.dm

```diff
@@ -125,3 +120,48 @@
 	max_integrity = 200
 	armor = list(MELEE = 20, BULLET = 20, LASER = 20, ENERGY = 0, BOMB = 20, BIO = 0, RAD = 0, FIRE = 30, ACID = 40)
 	custom_materials = list(/datum/material/gold = 10000)
+	foldabletype = /obj/item/wheelchair/gold
+
+/obj/item/wheelchair
+	name = "wheelchair"
+	desc = "A collapsed wheelchair that can be carried around."
+	icon = 'icons/obj/vehicles.dmi'
+	icon_state = "wheelchair_folded"
+	inhand_icon_state = "wheelchair_folded"
+	lefthand_file = 'icons/mob/inhands/items_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/items_righthand.dmi'
+	w_class = WEIGHT_CLASS_NORMAL
+	force = 8 //Force is same as a chair
+	var/unfolded_type = /obj/vehicle/ridden/wheelchair
+	custom_materials = list(/datum/material/iron = 10000)
+
+/obj/item/wheelchair/gold
+	name = "gold wheelchair"
+	desc = "A collapsed, shiny wheelchair that can be carried around."
+	icon = 'icons/obj/vehicles.dmi'
+	icon_state = "wheelchair_folded_gold"
+	inhand_icon_state = "wheelchair_folded_gold"
+	lefthand_file = 'icons/mob/inhands/items_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/items_righthand.dmi'
+	force = 10
+	unfolded_type = /obj/vehicle/ridden/wheelchair/gold
+
+/obj/vehicle/ridden/wheelchair/MouseDrop(over_object, src_location, over_location)  //Lets you collapse wheelchair
+	. = ..()
+	if(over_object == usr && Adjacent(usr) && can_fold)
```

Comment:
Instead of having a `can_fold` boolean (true/false value), can't we piggyback on the fact that `foldabletype` will always exist for things that can fold, and not for things that can't?